src/App.js

```diff
@@ -1,26 +1,21 @@
 import './App.css';
-import { Component } from 'react';
+// import { Component } from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import Container from 'react-bootstrap/Container'
 import Header from './Components/Header.js';
 import Main from './Components/Main.js';
 import Footer from './Components/Footer.js';
+import HornedBeast from './data.json';
 
 function App() {
   return  (
-    <div>
-      <Header />
-      <Main />
-      <Footer />
-      
-    </div>
+    <Container>
+      <Header title="Horned Animals"/>
+      <Main message="Make Your Beast Opinion Known" HornedBeast = {HornedBeast} />
```

Comment:
Just started lookin at this :)

- see prop of `HornedBeast`(change prop name to lowercase when ya can `hornedBeast={HornedBeast}